ext/standard/basic_functions.stub.php

```diff
@@ -2290,7 +2360,10 @@ function implode(string|array $separator, ?array $array = null): string {}
 /** @alias implode */
 function join(string|array $separator, ?array $array = null): string {}
 
-/** @refcount 1 */
+/**
+ * @refcount 1
+ * @compile-time-eval
+ */
```

Comment:
`@refcount` should be moved after `@compile-time-eval` for consistent order

Reply:
fixed, there were also a few inconsistnecies before this PR...